eng/pipelines/templates/stages/archetype-python-release.yml

```diff
@@ -34,7 +34,7 @@ stages:
                         Copy-Item $(Pipeline.Workspace)/${{parameters.ArtifactName}}/$underscorePrefix-[0-9]*.[0-9]*.[0-9]* $(Pipeline.Workspace)/${{artifact.safeName}}
                         Get-ChildItem $(Pipeline.Workspace)/${{artifact.safeName}}
                       displayName: Stage artifacts
-                    - template: ../../../common/pipelines/templates/steps/create-tags-and-git-release.yml
+                    - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
```

Comment:
Absolutely can appreciate this. I did not know it would resolve to the root of the repo like this! Awesome!